### ISSUE TITLE
Use loadScriptOnce instead of unmount script tag

### ISSUE THREAD

**Issue by evenchange4** [evenchange4/react-grecaptcha#69]
https://medium.com/@wonderboymusic/the-problem-with-html-in-react-and-graphql-6af9789352f6

**Comment by Khvalin** [comment]
yeah, that would be great. adding/ removing script tag on each mount/unmount seems to cause some undesirable side-effects
